src/components/App.tsx

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import Navigation from './Navigation';
 import Experiments from './Experiments';
+import CreateCanvas from './CreateCanvas'
+import ExperimentDetail from './ExperimentDetail';
+import { BrowserRouter as Router, Route } from 'react-router-dom'
+
 
 const App = () => {
   return (
-    <div className="App">
+    <Router>
       <Navigation />
-      <Experiments />
-    </div>
+      <Route exact path="/" component={ Experiments } />
+      <Route path="/up-only" component={ ExperimentDetail } />
+      <Route path="/canvas" component={ CreateCanvas } />
```

Comment:
Hey Salo, are we still using the lava background or just sticking with the gradient? If the latter, can we remove all the logic and files related to the lava background?

Reply:
since the lava option wasn't very popular, I think we'll stick with the gradient.. I'll remove it as well!